modules/codegen/src/main/scala/com/twilio/guardrail/swagger/SwaggerUtil.scala

```diff
@@ -38,18 +57,20 @@ object SwaggerUtil {
       }
     }
 
+    def lookupTypeName(clsName: String, tpeName: String, resolvedTypes: List[(String, Resolved)])(
+        f: Type => Type
+    ): Option[(String, Resolved)] =
+      resolvedTypes
+        .find(_._1 == tpeName)
+        .map(_._2.tpe)
+        .map(x => (clsName, Resolved(f(x), None, None)))
+
     def resolve_(values: List[(String, ResolvedType)]): Target[List[(String, Resolved)]] = {
       val (lazyTypes, resolvedTypes) = Foldable[List].partitionEither(values) {
         case (clsName, x: Resolved)         => Right((clsName, x))
         case (clsName, x: LazyResolvedType) => Left((clsName, x))
       }
 
-      def lookupTypeName(clsName: String, tpeName: String, resolvedTypes: List[(String, Resolved)])(f: Type => Type): Option[(String, Resolved)] =
```

Comment:
Why did the scoping of this function get expanded? It's not used anywhere else currently